src/components/ssl-observatory.js

```diff
@@ -611,7 +611,11 @@ SSLObservatory.prototype = {
     var HTTPSEverywhere = CC["@eff.org/https-everywhere;1"]
                             .getService(Components.interfaces.nsISupports)
                             .wrappedJSObject;
-    var win = channel ? HTTPSEverywhere.getWindowForChannel(channel) : null;
+    var win = null
+    if (channel) {
+      var browser = this.HTTPSEverywhere.getBrowserForChannel(channel);
+      var win = browser.contentWindow;
```

Comment:
This line gives `browser is null` ssl-observatory.js:624
Firefox 34.0.5 linux, https-everywhere latest master (currently this commit is latest: d278a6dff48e75fe6efc12df724f7a318ebbf3ac )
Seems to happen on browser startup(and restores tabs). I don't know what to make of it.